stats/release_queue.php

```diff
@@ -23,7 +23,8 @@
 $round = get_round_param($_GET, 'round_id', null, true);
 $name = @$_GET['name'];
 $listing_view_mode = get_enumerated_param($_GET, "show", "populated", array_keys($listing_view_modes));
-$unheld_only = get_integer_param($_GET, 'unheld_only', 0, 0, 1);
+// We should probably switch to get_bool_param but that would change our behavior.
```

Comment:
I don't think it's a problem to use `get_bool_param()` here.

Reply:
OK, updated.